scss/partials/_featured.scss

```diff
@@ -0,0 +1,54 @@
+/** TABLE OF CONTENTS - Cards
+ — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — /
+1 — Content Block
+
+/// END TABLE OF CONTENTS **/
+
+/** 1 - Featured Content
+------------------------------------------------------------*/
+.featured {
+  &__title {
+    @include vertical-spacing(md-2, bottom, margin);
+  }
+
+  .content-block {
```

Comment:
Something I'm debating here is whether or not this is BEM appropriate - nesting an already defined block inside another block. I know it's a case where the parent block is providing context that doesn't exist outside this. Let's go ahead and roll with this, but make a mental note to confirm the convention and possibly adjust it later.